src/java.naming/share/classes/com/sun/jndi/ldap/LdapPoolManager.java

```diff
@@ -395,47 +395,18 @@ private static void d(String msg, String o) {
         }
     }
 
-    private static final String getProperty(final String propName,
-        final String defVal) {
-        return AccessController.doPrivileged(
-            new PrivilegedAction<String>() {
-            public String run() {
-                try {
-                    return System.getProperty(propName, defVal);
-                } catch (SecurityException e) {
-                    return defVal;
-                }
-            }
-        });
+    private static final String getProperty(final String propName, final String defVal) {
+        PrivilegedAction<String> pa = () -> System.getProperty(propName, defVal);
+        return AccessController.doPrivileged(pa);
```

Comment:
Hmmm... This is not strictly equivalent but will work because java.naming is loaded by the boot loader and has the permission to read all system properties. I guess the code on the left-hand side was written at a time where JNDI was still in a stand-alone library?

Reply:
@dfuch I noticed the same about the swallowed exception that returns the default value. This wouldn't cause any regression?

Reply:
As I noted - the SecurityException should never occur so I don't expect this could cause a regression.